src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.wasm.cs

```diff
@@ -44,6 +44,18 @@ private void UpdateTextBoxView()
 			}
 		}
 
+		partial void OnSelectionChangedPartial()
```

Comment:
Judging from the tests (and from your comments). Wasm behavior was already correct, and tests don't capture changes in it. Why do we need any changes here at all?